app/common/data/interfaces/user.py

```diff
@@ -234,6 +234,10 @@ def get_usable_invitations_by_email(email: str) -> Sequence[Invitation]:
     ).all()
 
 
+def get_invitations_by_email(email: str) -> Sequence[Invitation]:
+    return db.session.scalars(select(Invitation).where(Invitation.email == email)).all()
```

Comment:
As a separate interface method this might want a very light touch integration test, or for less work it might be a named property on the method above 
```python
def get_invitations_by_email(email: str, *, usable: bool = False) -> Sequence[Invitation]:
```

I can see that there is a scenario in the page integration tests that checks the query param is set if there are remaining invitations which I think does indirectly cover this

Reply:
Refactored and test added